src/components/Config/Config.tsx

```diff
@@ -79,120 +78,57 @@ function Config() {
-                chainId: arbitrumOneChainId,
-                chainName: 'Arbitrum One',
-                rpcUrls: ['https://arb1.arbitrum.io/rpc'],
-                blockExplorerUrls: ['https://arbiscan.io/'],
-                nativeCurrency: {
-                  symbol: 'ETH', // 2-6 characters long
-                  decimals: 18,
-                },
-              },
-            ],
-          });
-        } catch (addError) {
-          console.log(addError);
-        }
-      }
-    } else {
-      notification.error({
-        message: (
-          <span>
-            Metamask is not installed. Metamask is required to connect Automate. Install the{' '}
-            <a href="https://metamask.io/download/" target="_blank" rel="noopener noreferrer">
-              Metamask extension.
-            </a>{' '}
-            If you have installed refresh the page.
-          </span>
-        ),
-      });
-    }
-  }, []);
-
-  const handleArbitrumConnection = useCallback(async () => {
+  const checkMetamaskInstalled = () => {
     const isMetamaskInstalled = typeof window.ethereum !== 'undefined';
-    if (isMetamaskInstalled) {
-      arbitrumNetworkConnect();
-    } else {
-      notification.error({
-        message: (
-          <span>
-            Metamask is not installed. Metamask is required to connect Automate. Install the{' '}
-            <a href="https://metamask.io/download/" target="_blank" rel="noopener noreferrer">
-              Metamask extension.
-            </a>{' '}
-            If you have installed refresh the page.
-          </span>
-        ),
-      });
+    if (!isMetamaskInstalled) {
+      throw notifications.metamaskNotInstalled();
     }
-  }, [arbitrumNetworkConnect]);
-
+  };
   const handleConnect = useCallback(async () => {
+    checkMetamaskInstalled();
     if (network === Network.Ethereum) {
-      handleEthereumConnection();
+      setSubmitted(true);
+      setCompleted(false);
     } else if (network === Network.Arbitrum) {
-      handleArbitrumConnection();
+      setSubmitted(true);
+      setCompleted(false);
+      setGasPriceAware(false);
+      setDraft(true);
+      setConfirmationTime(0);
     }
-  }, [handleArbitrumConnection, handleEthereumConnection, network]);
+  }, [network]);
 
   const handleCancel = useCallback(async () => {
     setSubmitted(false);
-  }, []);
+    if (network === Network.Arbitrum) {
+      setGasPriceAware(true);
+      setDraft(false);
+      setConfirmationTime(1);
+    }
+  }, [network]);
 
   const handleConfirmConfigured = useCallback(async () => {
     if (!(wallet.status === 'connected')) {
       await wallet.connect('injected');
     }
-    const isConnected = await checkConnection();
-    if (isConnected) {
-      notification.success({ message: `You're connected to Automate!` });
-      setSubmitted(false);
-      setCompleted(true);
+    const connectedNetwork = await checkConnection();
+    if (connectedNetwork !== 'notConnectedToAutomate') {
```

Comment:
Nested ifs again. Whenever that happens stop and think how this can be avoided. 99% of the time it's not a problem to refactor. You have an example with `checkMetamaskInstalled`